lib/logdna/resources.rb

```diff
@@ -14,4 +14,5 @@ module Resources
   ENDPOINT = "https://logs.logdna.com/logs/ingest"
   MAC_ADDR_CHECK = /^([0-9a-fA-F][0-9a-fA-F]:){5}([0-9a-fA-F][0-9a-fA-F])$/.freeze
   IP_ADDR_CHECK = /^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/.freeze
+  THREAD_POOL_SIZE = 5
```

Comment:
This isn't used, that I can see, unless this is a move.

Reply:
We can remove this for now

Reply:
Effectively renamed this to `MAX_CONCURRENT_REQUESTS` and am using it now